app/Http/Requests/AcceptSubmissionRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests;
+
+use App\Enums\UserTypes;
+use Illuminate\Foundation\Http\FormRequest;
+
+class AcceptSubmissionRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        $isDoctor = $this->user()->hasRole(UserTypes::DOCTOR->value);
```

Comment:
If you are using laravel-permissions package, you can manage authorization for roles via middleware. (More information here: https://spatie.be/docs/laravel-permission/v5/basic-usage/middleware)
This becomes particularly useful to avoid code repetition for the Role check in form requests.